Mathlib/Control/Monad/Cont.lean

```diff
@@ -184,10 +184,6 @@ instance [MonadCont m] [LawfulMonadCont m] : LawfulMonadCont (OptionT m) where
     ext; rfl
   callCC_dummy := by intros; simp only [callCC, OptionT.callCC, @callCC_dummy m _]; ext; rfl
 
-/- Porting note: In Lean 3, `One ω` is required for `MonadLift (WriterT ω m)`. In Lean 4,
```

Comment:
Why is this one safe to remove?

Reply:
Do we really care how `WriterT` was implemented in Lean 3? I think the current implementation works fine.